src/mgr/Mgr.cc

```diff
@@ -460,11 +462,12 @@ bool Mgr::ms_dispatch(Message *m)
       ceph_abort();
 
       py_modules.notify_all("mon_map", "");
+      m->put();
       break;
     case CEPH_MSG_FS_MAP:
       py_modules.notify_all("fs_map", "");
       handle_fs_map((MFSMap*)m);
-      m->put();
+      return false; // I shall let this pass through for Client
```

Comment:
if the `Client` is `client/Client.*`, then i am confused: Client does not share a messenger with Mgr. if not, then what Client it is?